metapackages/plugins-node-all/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I put this in metapackages to be consistent with where/how it was organized in opentelemetry-js. An argument could be made for nesting somewhere in plugins but I do see value in surfacing up it is a package of packages.

I'm open to suggestions if the current approach is not preferred (or we can always tweak later).